packages/ingest/test/test-http.js

```diff
@@ -14,22 +12,7 @@ const {
   headObject
 } = require('@cumulus/common/aws');
 const { randomString } = require('@cumulus/common/test-utils');
-
-class MyTestDiscoveryClass {
-  constructor(useList) {
-    this.decrypted = true;
-    this.path = '';
-    this.provider = {
-      protocol: 'http',
-      host: 'localhost',
-      port: 3030,
-      encrypted: false
-    };
-    this.useList = useList;
-  }
-}
-
-class MyTestHttpDiscoveryClass extends TestHttpMixin(MyTestDiscoveryClass) {}
+const HttpProviderClient = rewire('../HttpProviderClient');
```

Comment:
No unit tests for the `download` function.

Reply:
Agreed.